PublicMapping/districtbuilder
# Fix formatting of population counts

## Overview
Fix formatting of population counts by avoiding exponential notation

### Checklist


### Demo
#### Before
![Screenshot from 2020-10-14 13-06-30](https://user-images.githubusercontent.com/2926237/96021980-32432100-0e1e-11eb-8e78-d0232fcc1215.png)

#### After
![Screenshot from 2020-10-14 13-08-20](https://user-images.githubusercontent.com/2926237/96022107-5ef73880-0e1e-11eb-852e-8a94f4b88a4a.png)

### Notes
I added support for writing `manage` tests to be able to have a test for population count formatting. After doing so, it turned out that running the `client` tests also caused the `manage` tests to be run. ~I tried to turn off running both `manage` and `server` tests when running `client` tests by using a pattern in `testPathIgnorePatterns` like ` --testPathIgnorePatterns=src/{server,manage}` but that didn't work and then I also tried specifying the argument twice but that also didn't work. I wasn't able to specify that as a property of the `jest` property in `package.json` without ejecting (CRA said it was unsupported). All of that said, it seemed that no `server` tests were being run anyway, so I just switched turning off `server` tests for `manage` tests. This is not ideal, but it seems like we don't plan on writing `server` tests anytime soon so maybe it's okay.~ EDIT: This is fixed in a better way in ee655c0

I also ended up upgrading the TS version for `manage` which didn't seem to change much but made the new test deps happy (and is probably also a good thing to do in general). I'm not sure what the implications are of having different TS versions among `client`, `manage`, and `server` if we have shared types but I don't think it's a problem for right now and I figure some of this may change anyway when we do https://github.com/PublicMapping/districtbuilder/issues/179.

In terms of the actual fix, we are doing some rounding up (this should be evident in the tests; eg. `expect(abbreviateNumber(99500)).toEqual("100k");`) but that's how it was before, so I kept that the same. If we want to change it so we round down it should at least be easier to verify the behavior since now we have tests :tada: 

## Testing Instructions
* Re-process data for NY:
```
./scripts/manage process-geojson data/NY.geojson -o data/output-ny -n 12,4,4 -x 12,12,12
./scripts/manage publish-region data/output-ny US NY New\ York
```
* Turn on population labels
* Make sure the label for Putnam county looks right
* Click around and make sure population labels look right in general

Closes #419
